src/platform/forms-system/src/js/utilities/data/getFormDataFromSchemaId.js

```diff
@@ -0,0 +1,19 @@
+import get from 'lodash/get';
```

Comment:
Can we use the `get` utility function (`src/platform/utilities/data/get.js`) instead of the one from lodash?

Reply:
I did try to use that as my first approach, but oddly I get a lot of errors when I try to use that. maybe it has something to do with the fact that I'm trying to traverse both objects and arrays.